NitroxServer/Communication/Packets/Processors/BedEnterProcessor.cs

```diff
@@ -9,8 +9,8 @@ public class BedEnterProcessor : AuthenticatedPacketProcessor<BedEnter>
     {
         public override void Process(BedEnter packet, Player player)
         {
-            TimeKeeper timeKeeper = NitroxServiceLocator.LocateService<TimeKeeper>();
-            timeKeeper.SkipTime();
+            ScheduleKeeper scheduleKeeper = NitroxServiceLocator.LocateService<ScheduleKeeper>();
+            scheduleKeeper.ChangeTime("skip");
```

Comment:
`NitroxServiceLocator.LocateService<T>();` should not be used if possible. Use a constructor here with the `ScheduleKeeper` as parameter so that Autofac will inject the instance.